### PR TITLE
RESTFul Route added

### PR DESCRIPTION
**Pull request by psy2848048**
* Worked for balance query, transfer, bond, unbond.
* Temporarily, WASM binary comes from fixed path, will be revised after WASM storage feature is added.
* Run CLI tests if exists, but exclude from Codecov statistics (Regex doesn't work properly in `grep -v`)

### CODE REVIEW

**Comment by jhyeom26** [comment]
# [Codecov](https://codecov.io/gh/hdac-io/friday/pull/38?src=pr&el=h1) Report
> Merging [#38](https://codecov.io/gh/hdac-io/friday/pull/38?src=pr&el=desc) into [master](https://codecov.io/gh/hdac-io/friday/commit/7d357e4d8fc7f77386b4e5989e40a14291e0133c?src=pr&el=desc) will **decrease** coverage by `0.04%`.
> The diff coverage is `39.13%`.

[Impacted file tree graph](https://codecov.io/gh/hdac-io/friday/pull/38?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master      #38      +/-   ##
==========================================
- Coverage   50.74%   50.69%   -0.05%     
==========================================
  Files         318      319       +1     
  Lines       19896    19989      +93     
==========================================
+ Hits        10096    10134      +38     
- Misses       9081     9126      +45     
- Partials      719      729      +10
```


| [Impacted Files](https://codecov.io/gh/hdac-io/friday/pull/38?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [x/executionlayer/client/rest/rest.go](https://codecov.io/gh/hdac-io/friday/pull/38/diff?src=pr&el=tree#diff-eC9leGVjdXRpb25sYXllci9jbGllbnQvcmVzdC9yZXN0Lmdv) | `38.7% <39.13%> (ø)` | |
| [x/mock/app.go](https://codecov.io/gh/hdac-io/friday/pull/38/diff?src=pr&el=tree#diff-eC9tb2NrL2FwcC5nbw==) | `60.5% <0%> (+1.27%)` | :arrow_up: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/hdac-io/friday/pull/38?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/hdac-io/friday/pull/38?src=pr&el=footer). Last update [7d357e4...71904aa](https://codecov.io/gh/hdac-io/friday/pull/38?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by psy2848048** [comment]
Balance querying malfunctions.. WIP

**Comment by psy2848048** [comment]
The malfunction is fixed the PR https://github.com/hdac-io/friday/pull/39

File: x/executionlayer/client/rest/msgCreator.go [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+	}
+
+	// TODO: Change after WASM store feature merge
+	var bondingCode []byte
+	if bondIsTrue == true {
+		bondingCode = grpc.LoadWasmFile(os.ExpandEnv("$HOME/.nodef/contracts/bonding.wasm"))
+	} else {
+		bondingCode = grpc.LoadWasmFile(os.ExpandEnv("$HOME/.nodef/contracts/unbonding.wasm"))
+	}
+	bondingAbi := grpc.MakeArgsTransferToAccount(addr, req.BondedAmt)
```
Comment (psy2848048) [inline x/executionlayer/client/rest/msgCreator.go]: @JoowonYun Should we change here `addr` to public key too?

Comment (JoowonYun) [reply x/executionlayer/client/rest/msgCreator.go]: Yes, abi must to be use public key

**Pull request merged**